pkgs/by-name/in/interactive-html-bom/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  fetchFromGitHub,
+  kicad-small,
+  lib,
+  python3Packages,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "interactiveHtmlBom";
+  version = "2.9.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "openscopeproject";
+    repo = "InteractiveHtmlBom";
+    rev = "v${version}";
+    hash = "sha256-jUHEI0dWMFPQlXei3+0m1ruHzpG1hcRnxptNOXzXDqQ=";
+  };
+
+  build-system = [ python3Packages.hatchling ];
+
+  dependencies = [
+    python3Packages.jsonschema
+    python3Packages.wxpython
+    kicad-small
+  ];
+
```

Comment:
Are there any tests? If so please add them. At the very least add a versionCheckHook

Reply:
I don't think the project has tests, so we should set doCheck = false